spec/lib/guardian/flag_guardian_spec.rb

```diff
@@ -32,4 +32,11 @@
       flag.destroy!
     end
   end
+
+  describe "#can_toggle_flag?" do
+    it "returns true for admin and false for regular user" do
+      expect(Guardian.new(admin).can_toggle_flag?).to eq(true)
+      expect(Guardian.new(user).can_toggle_flag?).to eq(false)
```

Comment:
Would just add one more check for a `moderator` too so it's intentionally showing that we aren't allowing mods to do this